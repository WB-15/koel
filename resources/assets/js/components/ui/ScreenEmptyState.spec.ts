import { expect, it } from 'vitest'
import ComponentTestCase from '@/__tests__/ComponentTestCase'
import ScreenEmptyState from './ScreenEmptyState.vue'

new class extends ComponentTestCase {
  protected test () {
    it('renders', () => {
      expect(this.render(ScreenEmptyState, {
        slots: {
          icon: '<i class="my-icon"/>',
          default: 'Nothing here'
        }
      }).html()).toMatchSnapshot()
    })
  }
}
