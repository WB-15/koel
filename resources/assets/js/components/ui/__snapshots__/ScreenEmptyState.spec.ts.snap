// Vitest Snapshot v1

exports[`renders 1`] = `
<div class="screen-empty-state" data-testid="screen-empty-state">
  <div><span class="jumbo-icon"><i class="my-icon"></i></span>
    <div class="text">Nothing here</div>
  </div>
</div>
`;
